src/cpp/fastdds/builtin/type_lookup_service/TypeLookupManager.hpp

```diff
@@ -230,72 +160,47 @@ class TypeLookupManager
      */
     bool create_endpoints();
 
-    /**
-     * Get the RTPS participant
-     * @return RTPS participant
-     */
-    inline fastrtps::rtps::RTPSParticipantImpl* get_RTPS_participant()
-    {
-        return participant_;
-    }
-
-    //! Get out instanceName as defined in 7.6.2.3.4 of the XTypes 1.2 document
-    std::string get_instanceName() const;
-
     //! Aux method to send requests
     bool send_request(
-            TypeLookup_Request& req) const;
+            TypeLookup_Request& request) const;
 
     //! Aux method to send replies
     bool send_reply(
-            TypeLookup_Reply& rep) const;
+            TypeLookup_Reply& reply) const;
 
     //! Aux method to received requests
     bool recv_request(
             fastrtps::rtps::CacheChange_t& change,
-            TypeLookup_Request& req) const;
+            TypeLookup_Request& request) const;
 
     //! Aux method to received replies
     bool recv_reply(
             fastrtps::rtps::CacheChange_t& change,
-            TypeLookup_Reply& rep) const;
+            TypeLookup_Reply& reply) const;
+
+    /**
+     * Get the RTPS participant
+     * @return RTPS participant
+     */
+    inline fastrtps::rtps::RTPSParticipantImpl* get_RTPS_participant()
+    {
+        return participant_;
+    }
 
-    const fastrtps::rtps::GUID_t& get_builtin_request_writer_guid() const;
+    //! Get out instanceName as defined in 7.6.2.3.4 of the XTypes 1.2 document
+    std::string get_instanceName() const;
 
-    //!Pointer to the local RTPSParticipant.
```

Comment:
I am against removing code documentation, unless it is no longer applying. Instead of removing, I suggest improving.

Reply:
Restored in [7dafd25](https://github.com/eProsima/Fast-DDS/pull/4126/commits/7dafd25b2d97a27f2f53acf522b0d2c68e72a46e)